flatpak-flutter.yml

```diff
@@ -0,0 +1,56 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: io.github.friesi23.mhabit
+runtime: org.freedesktop.Sdk
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm19
+command: mhabit
+separate-locales: false
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=system-bus
+  - --device=dri
+  - --talk-name=org.freedesktop.portal.Desktop
```

Comment:
this is NEVER necesary, not allowed. Not sure where the documentation says to do that.